test/generator/template_values_test.rb

```diff
@@ -56,4 +56,115 @@ def teardown
       $LOAD_PATH.delete 'test/fixtures/xruby/lib'
     end
   end
+
+  class CaseValuesTest < Minitest::Test
+    def setup
+      $LOAD_PATH.unshift 'test/fixtures/xruby/lib'
```

Comment:
Modifying the load path for every test should not be necessary.
But there might be something else to do so that everything still works.